lib/tapioca/compilers/dsl/active_record_associations.rb

```diff
@@ -44,6 +46,9 @@ module Dsl
       #     sig { params(value: T.nilable(::User)).void }
       #     def author=(value); end
       #
+      #     sig { params(attributes: T.untyped).returns(T.untyped) }
+      #     def author_attributes=(attributes); end
```

Comment:
The doc says:

> The attribute writer is named after the association, which means that in the following example, two new methods are added to your model: `author_attributes=(attributes)` and `pages_attributes=(attributes)`

Shouldn't we see two methods being added to the model? I only see `=` methods in this example.

Reply:
`accepts_nested_attributes_for :author, :pages` adds `author_attributes=` and `pages_attributes=` (note that two names are passed to `accepts_nested_attributes_for`: `:author` and `:pages`). One writer method for each association that has a nested attribute.

In this example case, we have 3 writers added since we called the method with associations.

Reply:
Urgh I don't know how to read anymore...